src/OpenSSL/SSL.py

```diff
@@ -680,31 +680,24 @@ class Context:
     """
 
     _methods = {
-        SSLv2_METHOD: "SSLv2_method",
```

Comment:
If we're going to remove this here (which I'm fine with), then we probably want to remove the top level constants too? The latter is an instantly breaking change for people who import it, while the current approach breaks at runtime.

I do wonder how many people will tell us "I use pyopenssl to do bad TLS deliberately"...?

Reply:
Done